ddtrace/opentracer/tracer_test.go

```diff
@@ -6,10 +6,14 @@
 package opentracer
 
 import (
+	"context"
 	"testing"
 
+	"github.com/opentracing/opentracing-go"
```

Comment:
This import goes down there, with `assert` and the rest of the external imports.